db/write_thread.cc

```diff
@@ -406,9 +410,9 @@ size_t WriteThread::EnterAsBatchGroupLeader(Writer* leader,
   // Allow the group to grow up to a maximum size, but if the
   // original write is small, limit the growth so we do not slow
   // down the small write too much.
-  size_t max_size = 1 << 20;
-  if (size <= (128 << 10)) {
-    max_size = size + (128 << 10);
+  size_t max_size = upper_limit_write_batch_group_size_bytes;
+  if (size <= lower_limit_write_batch_group_size_bytes) {
```

Comment:
We always try to minimize options. Do we need both of upper and lower for now?
Can we say users only take upper limit, and the lower limit is going to be 1/8 of that?

Reply:
Good suggestion. Updated options to allow configuration for just the max limit. The lower limit will be 1/8